api/schema/book.py

```diff
@@ -7,6 +7,7 @@ class BookBase(BaseModel):
     author: str
     publication_year: Optional[int] = None
     rating: Optional[int] = None
+    genre: str
```

Comment:
There are benefits to using a string versus an enum here. For a quick implementation, string is fine! I would recommend making this field optional, however.